File: app/src/main/java/com/example/jetpackmvvm/sample/util/BindingAdapters.java
package com.example.jetpackmvvm.sample.util;

import android.content.Context;
import android.widget.ImageView;

import androidx.databinding.BindingAdapter;

import com.bumptech.glide.Glide;

public class BindingAdapters {

    @BindingAdapter("imageUrl")
    public static void bindImage(ImageView imageView, String url) {
        Glide.with(imageView.getContext())
                .load(url)
                .into(imageView);
    }
}

File: app/src/main/java/com/terry/jetpackmvvm/sample/util/BindingAdapters.kt
package com.terry.jetpackmvvm.sample.util

import android.widget.ImageView
import androidx.databinding.BindingAdapter
import com.terry.jetpackmvvm.sample.GlideApp
import com.terry.jetpackmvvm.sample.util.image.ImageUtils

object BindingAdapters {
    @JvmStatic
    @BindingAdapter("imageUrl")
    fun bindImage(imageView: ImageView, url: String) {
        ImageUtils.loadRoundImage(imageView, url, 8)
    }
}